cmd/start.go

```diff
@@ -14,7 +17,7 @@ func startCommand() *cobra.Command {
 		startCmd = cobra.Command{
 			Use:   "start",
 			Short: "Starts the Timelock Worker daemon",
-			Run:   startHandler,
+			Run:   startHandlerAndServer,
```

Comment:
The name `startHandler` is a convention on how to use cobra. I'd leave this as startHandler.